app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -12,7 +12,7 @@ CONFIG_COMP_MULTIBAND_DRC=y
 CONFIG_PM=y
 CONFIG_PM_DEVICE=y
 CONFIG_PM_DEVICE_RUNTIME=y
-CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=n
+CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=y
 CONFIG_PM_DEVICE_POWER_DOMAIN=y
 CONFIG_PM_POLICY_CUSTOM=y
 
```

Comment:
And this as well:
```
Until the IPC driver is reworked to enable normal initialization without
additional conditions we need to manually trigger suspend and resume
actions.
```

Does this cover context-save only, or some more general problem? What are the "additional conditions"?